package.json

```diff
@@ -153,15 +153,15 @@
     "@cdktf/hcl2json": "0.19.0",
     "@iarna/toml": "3.0.0",
     "@opentelemetry/api": "1.6.0",
-    "@opentelemetry/context-async-hooks": "1.17.1",
-    "@opentelemetry/exporter-trace-otlp-http": "0.44.0",
-    "@opentelemetry/instrumentation": "0.44.0",
+    "@opentelemetry/context-async-hooks": "1.17.0",
+    "@opentelemetry/exporter-trace-otlp-http": "0.43.0",
```

Comment:
you should only revert the 0.44.0 versions or only this package, as only here is a bug.